### PR TITLE
Split LVs by mutation to test for significance.

### PR DESCRIPTION
**Pull request by sgosline**
Somehow over-incremented, but 17 has always been my lucky number so 🤷‍♀

### CODE REVIEW

File: results/17-LV-GenomicVariantTest.Rmd [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+```{r,warning=FALSE}
+wgs.vars=synTableQuery("SELECT Hugo_Symbol,Protein_position,specimenID,IMPACT,FILTER,ExAC_AF FROM syn20551862")$asDataFrame()
+exome.vars=synTableQuery("SELECT Hugo_Symbol,Protein_position,specimenID,IMPACT,FILTER,ExAC_AF FROM syn20554939")$asDataFrame()
+
+all.vars<-rbind(select(wgs.vars,'Hugo_Symbol','Protein_position','specimenID','IMPACT','ExAC_AF'),
+    select(exome.vars,'Hugo_Symbol','Protein_position','specimenID','IMPACT','ExAC_AF'))%>%
+  subset(ExAC_AF<0.05)
+
+
+fn <- tempfile(pattern = "", fileext = ".feather")
```
Comment (allaway) [inline results/17-LV-GenomicVariantTest.Rmd]: this is now available as a table here: https://www.synapse.org/#!Synapse:syn21046991/tables/

If you update the code to include this, then I can add CHOP pNFs and it should be easy to rerun this Rmd and get the new data

File: results/17-LV-GenomicVariantTest.Rmd [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+  }
+#}
+
+
+```
+
+
+
+## Breaking down by tumor type
+At first glance it seems that a lot of these are separating out cNFs (i.e. mast cell signaling) from other types. However, I'm getting the same error I get in notebook number 11, so am unsure about how to proceed.
```
Comment (allaway) [inline results/17-LV-GenomicVariantTest.Rmd]: what error? 

And do you think the separation out by cNF might be due to WGS vs WES in the other samples?

File: results/17-LV-GenomicVariantTest.Rmd [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+- [Latent Variable Feather File](https://www.synapse.org/#!Synapse:syn21046991/tables/)
+
+For this data we are also using any data for which there are gene variants (cNFs, pNFs, MPNSTs):
+- [Exome-Seq variants](https://www.synapse.org/#!Synapse:syn20554939/tables/)
+- [WGS Variants](https://www.synapse.org/#!Synapse:syn20551862/tables/) 
+
+Let's see if there are any LVs that split based on gene variant. Because we're having trouble scaling with the number of latent variables, I only look at variants that occur in less than 5% of the population. notice this is a difference from notebook #11.
+
+```{r,warning=FALSE}
+wgs.vars=synTableQuery("SELECT Hugo_Symbol,Protein_position,specimenID,IMPACT,FILTER,ExAC_AF FROM syn20551862")$asDataFrame()
```
Comment (jaybee84) [inline results/17-LV-GenomicVariantTest.Rmd]: It seems gnomAD scores have more coverage than ExAC_AF (https://www.genomeweb.com/sequencing/gnomad-resource-introduced-ashg-meeting-doubles-exac-dataset). We should probably switch to using using gnomAD as our metric.

File: results/17-LV-GenomicVariantTest.Rmd (hunk 2 above)
Comment (sgosline) [reply results/17-LV-GenomicVariantTest.Rmd]: The error is due to incomplete coverage of variants withins pecicic tumor types, I think. A `trycatch` would be my defacto way of handling it.

File: results/17-LV-GenomicVariantTest.Rmd (hunk 3 above)
Comment (sgosline) [reply results/17-LV-GenomicVariantTest.Rmd]: ok, i still used a threshold of <0.05, does this make sense?

File: results/17-LV-GenomicVariantTest.Rmd (hunk 1 above)
Comment (sgosline) [reply results/17-LV-GenomicVariantTest.Rmd]: fixed.

File: results/17-LV-GenomicVariantTest.Rmd (hunk 3 above)
Comment (jaybee84) [reply results/17-LV-GenomicVariantTest.Rmd]: I have used < 0.01 (albeit ExAC) in the earlier analyses given that some papers used that as their filtering threshold for common variants... 0.05 as threshold should work too if we can cite why we used that threshold.

**Comment by allaway** [comment]
LGTM - I think @jaybee84 has to approve too now :)

**Pull request merged**